rules/private/copy_file_private.bzl

```diff
@@ -128,27 +123,20 @@ def copy_file(name, src, out, is_executable = False, allow_symlink = False, **kw
           handle symlinks (which most UNIX tools can).
       **kwargs: further keyword arguments, e.g. `visibility`
     """
+
+    copy_file_impl = _copy_file
     if is_executable:
```

Comment:
Silly github UI won't let me comment  on line 96. There is no more _ximpl.

Reply:
I *think* this was already fixed in 3f0948767e476c0b301caa95ab7f861a8314fcb1.